src/sentry/static/sentry/app/components/commitAuthorStats.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+import LoadingIndicator from '../components/loadingIndicator';
+import LoadingError from '../components/loadingError';
+import Avatar from '../components/avatar';
+
+import TooltipMixin from '../mixins/tooltip';
+import ApiMixin from '../mixins/apiMixin';
+
+const CommitBar = React.createClass({
+  propTypes: {
+    whole: React.PropTypes.number.isRequired,
```

Comment:
You can leave them as is, but I'd add those comments (the ones you just wrote) next to them.